src/spatial/spatial_massaction_jump.jl

```diff
@@ -44,89 +44,113 @@ end
-                                                      B <: AMatOrNothing}
+    useiszero = true,
+    nocopy = false) where {A <: AVecOrNothing,
+    B <: AMatOrNothing}
     SpatialMassActionJump(urates, srates, rs, ns, nothing; scale_rates = scale_rates,
-                          useiszero = useiszero, nocopy = nocopy)
+        useiszero = useiszero, nocopy = nocopy)
 end
 
 function SpatialMassActionJump(srates::B, rs, ns, pmapper; scale_rates = true,
-                               useiszero = true,
-                               nocopy = false) where {B <: AMatOrNothing}
+    useiszero = true,
+    nocopy = false) where {B <: AMatOrNothing}
     SpatialMassActionJump(nothing, srates, rs, ns, pmapper; scale_rates = scale_rates,
-                          useiszero = useiszero, nocopy = nocopy)
+        useiszero = useiszero, nocopy = nocopy)
 end
 function SpatialMassActionJump(srates::B, rs, ns; scale_rates = true, useiszero = true,
-                               nocopy = false) where {B <: AMatOrNothing}
+    nocopy = false) where {B <: AMatOrNothing}
     SpatialMassActionJump(nothing, srates, rs, ns, nothing; scale_rates = scale_rates,
-                          useiszero = useiszero, nocopy = nocopy)
+        useiszero = useiszero, nocopy = nocopy)
 end
 
 function SpatialMassActionJump(urates::A, rs, ns, pmapper; scale_rates = true,
-                               useiszero = true,
-                               nocopy = false) where {A <: AVecOrNothing}
+    useiszero = true,
+    nocopy = false) where {A <: AVecOrNothing}
     SpatialMassActionJump(urates, nothing, rs, ns, pmapper; scale_rates = scale_rates,
-                          useiszero = useiszero, nocopy = nocopy)
+        useiszero = useiszero, nocopy = nocopy)
 end
 function SpatialMassActionJump(urates::A, rs, ns; scale_rates = true, useiszero = true,
-                               nocopy = false) where {A <: AVecOrNothing}
+    nocopy = false) where {A <: AVecOrNothing}
     SpatialMassActionJump(urates, nothing, rs, ns, nothing; scale_rates = scale_rates,
-                          useiszero = useiszero, nocopy = nocopy)
+        useiszero = useiszero, nocopy = nocopy)
 end
 
 function SpatialMassActionJump(ma_jumps::MassActionJump{T, S, U, V}; scale_rates = true,
-                               useiszero = true, nocopy = false) where {T, S, U, V}
+    useiszero = true, nocopy = false) where {T, S, U, V}
     SpatialMassActionJump(ma_jumps.scaled_rates, ma_jumps.reactant_stoch,
-                          ma_jumps.net_stoch, ma_jumps.param_mapper;
-                          scale_rates = scale_rates, useiszero = useiszero, nocopy = nocopy)
+        ma_jumps.net_stoch, ma_jumps.param_mapper;
+        scale_rates = scale_rates, useiszero = useiszero, nocopy = nocopy)
 end
 
 ##############################################
 
-function get_num_majumps(smaj::SpatialMassActionJump{Nothing, Nothing, S, U, V}) where
+function get_num_majumps(smaj::SpatialMassActionJump{
```

Comment:
@ChrisRackauckas this kind of reformatting is exactly what we discussed in that thread, which break VSCode functionality, and has readability issues. It is why we haven't reformatted while awaiting the fix to that thread, since it seems like this is a non-reversible format change (i.e. the formatter itself won't fix this in the future).